backend.native/tests/runtime/basic/simd.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package runtime.basic.simd
+
+import kotlin.test.*
+
+
+@Test fun runTest() {
+    testSetGet()
+    testString()
+    testOOB()
+    testEquals()
+    testHash()
+}
+
+fun testSetGet() {
+    var v4any = vectorOf(0, 1, 2, 3)
+    (0 until 4).forEach { assertEquals(it, v4any.getIntAt(it)) }
+
+    // type punning: set the variable to another runtime type
+    val a = arrayOf(1f, 3.162f, 10f, 31f)
+    v4any = vectorOf(a[0], a[1], a[2], a[3])
+    (0 until 4).forEach { assertEquals(a[it], v4any.getFloatAt(it)) }
+}
+
+fun testString() {
+    val v4i = vectorOf(100, 1024, Int.MAX_VALUE, Int.MIN_VALUE)
+    assertEquals("(0x64, 0x400, 0x7fffffff, 0x-80000000)", v4i.toString())
```

Comment:
`0x-80000000` is somewhat incorrect.

Reply:
Looks ugly indeed. But this is what Int.toString(16) does. Let's have it fixed first as the assertion is aligned with that behavior.